README.md

```diff
@@ -34,146 +38,106 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
 
-   ```bash
-   echo "Hello Rook!" > /tmp/rookFS/hello
-   cat /tmp/rookFS/hello
-   ```
+For a quick start, use Rook fork for [coreos-kubernetes](https://github.com/rook/coreos-kubernetes). This will bring up a multi-node Kubernetes cluster, configured for using the rbd volume plugin.
 
-5. Unmount the shared file system (this does **not** delete the data from the cluster)
+```
+$ git clone https://github.com/rook/coreos-kubernetes.git
+$ cd coreos-kubernetes/multi-node/vagrant
+$ vagrant up
+$ export KUBECONFIG="$(pwd)/kubeconfig"
+$ kubectl config use-context vagrant-multi
+```
 
-   ```bash
-   ./rook filesystem unmount --path /tmp/rookFS
-   ```
+Then wait for the cluster to come up and verify that kubernetes is done initializing (be patient, it takes a bit):
 
-6. Cleanup the shared file system from the cluster (this **does** delete the data from the cluster)
+```
+$ kubectl cluster-info
+```
 
-   ```
-   ./rook filesystem delete --name testFS
-   ```
+If you see a url response, you are ready to go.
 
-#### Object Storage
-1. Create an object storage instance in the cluster
+#### Deploy Rook
 
-   ```bash
-   ./rook object create
-   ```
+Rook can be setup and deployed in Kubernetes by simply deploying the [rook-operator](demo/kubernetes/rook-operator.yaml) deployment manifest.
+You will find this manifest and all our example manifest files in the [demo/kubernetes](demo/kubernetes) folder.
 
-2. Create an object storage user
+```
+$ cd demo/kubernetes
+$ kubectl create namespace rook
+$ kubectl create -f rook-operator.yaml
+```
 
-   ```bash
-   ./rook object user create rook-user "A rook rgw User"
-   ```
+Use `kubectl` to list pods in the rook namespace. You should be able to see the following: 
 
-3. Get the connection information for accessing object storage
+```
+$ kubectl -n rook get pod
+NAME                            READY     STATUS    RESTARTS   AGE
+mon0                            1/1       Running   0          1m
+mon1                            1/1       Running   0          1m
+mon2                            1/1       Running   0          1m
+osd-n1sm3                       1/1       Running   0          1m
+osd-pb0sh                       1/1       Running   0          1m
+osd-rth3q                       1/1       Running   0          1m
+rgw-1785797224-9xb4r            1/1       Running   0          1m
+rgw-1785797224-vbg8d            1/1       Running   0          1m
+rook-api-4184191414-l0wmw       1/1       Running   0          1m
+rook-operator-349747813-c3dmm   1/1       Running   0          1m
+```
 
-   ```bash
-   eval $(./rook object connection rook-user --format env-var)
-   ```
+#### Provision Storage
+Before Rook can start provisioning storage, a StorageClass needs to be created. This is used to specify the storage privisioner, parameters, admin secret and other information needed for Kubernetes to interoperate with Rook for provisioning persistent volumes.
```

Comment:
i'd suggest removing the wording like "Before Rook can start" and "before we proceed" in the next paragraph. We're already planning on proceeding, so just tell me what to do instead of what to do before i proceed. :)